### PR TITLE
chore: configure operation store index

### LINKED ISSUE #236: Configure index for operation store
Configure an index for operation store in order to optimize query time.

### PR DESCRIPTION
**Pull request by sandrask**
Configure operation store index

Closes #236

Signed-off-by: Sandra Vrtikapa <sandra.vrtikapa@securekey.com>

### CODE REVIEW

**Review by fqutishat — APPROVED** [review]

File: pkg/store/operation/store.go [hunk 1]
```diff
@@ -17,24 +17,66 @@ import (
...
 
 // New creates new operation store.
 func New(provider storage.Provider) (*Store, error) {
-	store, err := provider.OpenStore(nameSpace)
+	store, err := provider.OpenStore(namespace)
 	if err != nil {
 		return nil, fmt.Errorf("failed to open operation store: %s", err.Error())
 	}
 
+	config, err := provider.GetStoreConfig(namespace)
```
Comment (DRK3) [inline pkg/store/operation/store.go:33]: The SetStoreConfig logic in the store implementation actually already handles checking for an existing store config. I believe you can simply set the store config here (it won't throw an error).

(Currently CouchDB is the only one with true Store Configuration management, but I plan on keeping the behaviour I described above consistent among store implementations)

Comment (sandrask) [reply pkg/store/operation/store.go:33]: It throws an error "CouchDB: Internal Server Error: Encountered a conflict while saving the design document.", that's why I did all this error checking.

Comment (DRK3) [reply pkg/store/operation/store.go:33]: Ah - I will fix this internally. I can update this code later once it's no longer needed.

**Review by DRK3 — APPROVED** [review]

File: pkg/store/operation/store.go (hunk 1 above)
Comment (sandrask) [reply pkg/store/operation/store.go:33]: Sounds good

**Pull request merged**